pynvml/nvml.py

```diff
@@ -654,11 +699,31 @@
 NVML_GRID_LICENSE_FEATURE_CODE_COMPUTE      = 4
 
 _nvmlGridLicenseExpiryStatus_t = c_uint8
-NVML_GRID_LICENSE_EXPIRY_NOT_AVAILABLE    = 0   # Expiry information not available
-NVML_GRID_LICENSE_EXPIRY_INVALID          = 1   # Invalid expiry or error fetching expiry
-NVML_GRID_LICENSE_EXPIRY_VALID            = 2   # Valid expiry
-NVML_GRID_LICENSE_EXPIRY_NOT_APPLICABLE   = 3   # Expiry not applicable
-NVML_GRID_LICENSE_EXPIRY_PERMANENT        = 4   # Permanent expiry
+NVML_GRID_LICENSE_EXPIRY_NOT_AVAILABLE    = 0,   # Expiry information not available
+NVML_GRID_LICENSE_EXPIRY_INVALID          = 1,   # Invalid expiry or error fetching expiry
+NVML_GRID_LICENSE_EXPIRY_VALID            = 2,   # Valid expiry
+NVML_GRID_LICENSE_EXPIRY_NOT_APPLICABLE   = 3,   # Expiry not applicable
+NVML_GRID_LICENSE_EXPIRY_PERMANENT        = 4,   # Permanent expiry
```

Comment:
```suggestion
NVML_GRID_LICENSE_EXPIRY_NOT_AVAILABLE    = 0   # Expiry information not available
NVML_GRID_LICENSE_EXPIRY_INVALID          = 1   # Invalid expiry or error fetching expiry
NVML_GRID_LICENSE_EXPIRY_VALID            = 2   # Valid expiry
NVML_GRID_LICENSE_EXPIRY_NOT_APPLICABLE   = 3   # Expiry not applicable
NVML_GRID_LICENSE_EXPIRY_PERMANENT        = 4   # Permanent expiry
```

More `tuple` vs `int` discrepancies here. Pretty sure these are C -> python conversion bugs.